benchmarks/ASQA/README.md

```diff
@@ -0,0 +1,40 @@
+# ASQA BENCHMARK
+
+This benchmark is designed to evaluate the performance of the [ASQA dataset](https://huggingface.co/datasets/din0s/asqa). For `generate.py`, we followed [FLARE](https://github.com/jzbjyb/FLARE), using `gpt-3.5-turbo-instruct` with no retrieval settings.
+
+## Usage
+
+1. Prepare your model output results in the `output` directory. You can just attatch your `answers` to the end of each example in the origin ASQA dataset as what we did in the `dataset.json` file.
+2. Run `run.sh` to start dataset generation. The result will be saved in `output` directory, named `results.json`. And the `result_datasets.json` file is the json dump of the detailed result.
+
```

Comment:
1. Upload dataset.json file to HuggingFace and provide a link here. 
2. Maybe its better to seperate dataset generation step and evaluate step.